src/tests/stub_libmgr.cc

```diff
@@ -233,7 +233,7 @@ bool Mgr::StoreToCommWriter::doneAll() const STUB_RETVAL(false)
 void Mgr::StoreToCommWriter::scheduleStoreCopy() STUB
 void Mgr::StoreToCommWriter::noteStoreCopied(StoreIOBuffer ioBuf) STUB
 void Mgr::StoreToCommWriter::NoteStoreCopied(void* data, StoreIOBuffer ioBuf) STUB
-void Mgr::StoreToCommWriter::Abort(void* param) STUB
+void Mgr::StoreToCommWriter::HandleStoreAbort(StoreToCommWriter *param) STUB
```

Comment:
Please remove the unused "param" name in touched stub lines.

```suggestion
void Mgr::StoreToCommWriter::HandleStoreAbort(StoreToCommWriter *) STUB
```

Reply:
Done (175afa2).